experimental/traffic-portal/src/app/api/delivery-service.service.ts

```diff
@@ -158,6 +159,16 @@ export class DeliveryServiceService extends APIService {
 		this.deliveryServiceTypes = new Array<Type>();
 	}
 
+	/**
+	 * Gets a list of all Steering Configurations
+	 *
+	 * @returns An array of Steering Configurations
+	 */
+	public async getSteering(): Promise<Array<SteeringConfiguration>> {
+		const path = "steering";
+		return this.get<[SteeringConfiguration]>(path).toPromise();
```

Comment:
This type argument is mismatched with the return type. Either it should be an array of ambiguous length, or the return type should be a tuple of length 1.